Makefile

```diff
@@ -6,31 +6,35 @@
 
 #
 # Copyright (c) 2019, Joyent, Inc.
+# Copyright 2023 MNX Cloud, Inc.
 #
 
 NAME=rabbitmq
 TAR = tar
 RELEASE_TARBALL=$(NAME)-pkg-$(STAMP).tar.gz
 
-BASE_IMAGE_UUID = fd2cc906-8938-11e3-beab-4359c665ac99
+# triton-origin-x86_64-21.4.0
+BASE_IMAGE_UUID = 502eeef2-8267-489f-b19c-a206906f57ef
 BUILDIMAGE_NAME = $(NAME)
-BUILDIMAGE_DESC	= SDC RabbitMQ
+BUILDIMAGE_DESC	= Triton RabbitMQ
 BUILDIMAGE_PKGSRC = \
-	perl-5.14.2nb3 \
-	iodbc-3.52.7 \
-	libffi-3.0.9nb1 \
-	python27-2.7.2nb2 \
-	py27-setuptools-0.6c11nb1 \
-	py27-simplejson-2.1.1 \
-	erlang-14.1.4nb1 \
-	rabbitmq-2.7.1
+	perl-5.34.0nb3 \
```

Comment:
Not a code nit, but a bug-report nit.  Would appreciate seeing in the ticket why versions are being picked IF AND ONLY IF they aren't just the "this is the latest/greatest/default in 21.4 pkgsrc."  `rabbitmq271-2.7.1` e.g.

Also make sure test notes are in ticket.

Reply:
In most cases these are the *only* versions in pkgsrc. For erlang and rabbitmq, the versions are the ones ported forward from the 2011 image.